### DISCUSSION TITLE
Not possible to get Student Benefits, Auto Denied

### DISCUSSION THREAD

**Discussion by FuscusLucian** [community/community#52613] [GitHub Education]
This GitHub account is registered with the e-mail that was provided by my University, it is directly affiliated to Åland's University Of Applied Sciences in Finland.

I provided two documentation papers confirming my education here, one of which has my full name, address and other personal information. The paper is in Swedish (I live in the Swedish Speaking part of Finland) , so maybe the document not being in English might be the cause ?

My github account has all of the set up requirement it asked for, i have add blocks turned off and allowed the site to grab my location.

All of this and it denied me twice in a row instantly.

**The response i get from GitHub :**
Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

**Comment by samus-aran** [comment]
Hi @FuscusLucian 👋 

Let's see how we can assist ⭐ 

First and foremost you will need to enable browser location. In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program.

You will have the most success applying if you are able to do so from your school’s campus.

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are still having difficulties enabling your location check out [this discussion on assessing your hardware or operating system's settings.](https://github.com/orgs/community/discussions/42246)
- If you are using a VPN, please disable it when you reapply. 
- If you are not using a VPN, try reapplying when connected to the internet from a different network.

The next rejection reason is a common pre-processing warning displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Hope this helps! Let us know how you get along 👍

**Comment by FuscusLucian** [comment]
Well i attempted it again, just auto-denies now because i already gave it
my University papers which it deemed not sufficient so it auto denies them
as soon as i show them, despite papers showing i got into the uni, how long
my 1st year of studying has been so far and the total study time for the
remaining 3 years.

I will attempt this I suppose from the actual Uni building next time
tomorrow again.
It is not asking for my location any longer, it only asked me once the
first time and I approved it to take my geo location position, I did not
have any issues with that.
I guess I'll see how it reacts when I attempt this same process the
following day via my laptop at the university.
